cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -59,8 +59,181 @@ def test_dropdownlist_required_dialog_element(appliance, catalog_item):
     Polarion:
         assignee: nansari
         initialEstimate: 1/4h
+        testtype: functional
         casecomponent: Services
+        startsin: 5.10
+        tags: service
     """
     service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
     view = navigate_to(service_catalogs, 'Order')
     assert view.submit_button.disabled
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_text_box_required_dialog_element():
+    """ Tests service text_box dialog required element
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/4h
+        startsin: 5.10
+        tags: service
+        testSteps:
+            1. Create a dialog .set required true to text box
+            2. Use the dialog in a catalog .
```

Comment:
Fix `.set` typo here and in test steps below, unless this is intentional? Also typo with `Use the dialog in a catalog.`